node/src/node_configurator/unprivileged_parse_args_configuration.rs

```diff
@@ -222,7 +222,11 @@ pub fn make_neighborhood_config<T: UnprivilegedParseArgsConfiguration + ?Sized>(
     };
 
     match make_neighborhood_mode(multi_config, neighbor_configs, persistent_config) {
-        Ok(mode) => Ok(NeighborhoodConfig { mode, min_hops }),
+        Ok(mode) => Ok(NeighborhoodConfig {
+            mode,
+            min_hops,
+            country: "ZZ".to_string(),
```

Comment:
Why does this always say ZZ. It's hardcoded. Why do you need this field actually? You know about a place where we later modify it to something else? Even if there was such a weird place then this should have been an option. 

I do think this is wrong. Probably a needless field.